app/src/main/java/fr/neamar/kiss/result/SearchResult.java

```diff
@@ -32,9 +32,16 @@ public View display(Context context, int position, View v) {
 
         TextView appName = (TextView) v.findViewById(R.id.item_search_text);
         String text = context.getString(R.string.ui_item_search);
```

Comment:
Please move this into the `else` branch of `if(searchPojo.direct)` to make it clearer (code flow-wise) where this is (and isn't) used.